### ISSUE TITLE
Yeta wrench causing crashes

### ISSUE THREAD

**Issue by IcepickJosh** [MJRLegends/Space-Astronomy-Feedback-#121]
I am experiencing a crash when crafting an Ender IO yeta wrench. The crash occurs after I pull the wrench out of the crafting grid an drop it into my inventory.

In an attempt to validate this, I duplicated this crash by creating a new world and using NEI to get the necessary crafting components without any other machines or structures in the world.

I'm currently running version 1.3.0 of Space Astronomy.

[EnderIO_crash_log.txt](https://github.com/MJRLegends/Space-Astronomy-Feedback-/files/46426/EnderIO_crash_log.txt)

**Comment by phattpks** [comment]
I am having  similar issue with Yea wrench. Single player crashes when yeta wrench is put into inventory. On my server client crashes on load up. I removed my player.dat file and can log in fine untile I put a yeta wrench in inventory. I have uninstalled and reinstalled client aswell with no fix.

I am on 1.3.0 aswell

[crash-2015-11-28_14.45.33-client.txt](https://github.com/MJRLegends/Space-Astronomy-Feedback-/files/46430/crash-2015-11-28_14.45.33-client.txt)
